src/app/chip_data_model.gni

```diff
@@ -256,15 +258,15 @@ template("chip_data_model") {
           "${_app_root}/clusters/${cluster}/${cluster}.h",
           "${_app_root}/clusters/${cluster}/operational-state-delegate.h",
         ]
+      } else if (cluster == "icd-management-server") {
+        sources += [ "${_app_root}/clusters/${cluster}/${cluster}.cpp" ]
+
+        public_deps += [ "${chip_root}/src/app/icd:monitoring-table" ]
```

Comment:
Is this meant to go into `public_deps`?  Or `deps`?  Specifically, is the intent that if you have data_model as a dep then you can include IcdMonitoringTable.h without explicitly saying you depend on icd:monitoring-table?

I would think it should go into `deps`.

Reply:
You are correct, I changed monitoring-table to be a `deps` in both instances